ql/src/test/results/clientpositive/encrypted/encryption_move_tbl.q.out

```diff
@@ -109,8 +109,8 @@ POSTHOOK: query: SHOW TABLES
 POSTHOOK: type: SHOWTABLES
 POSTHOOK: Input: database:default
 src
-FAILED: SemanticException [Error 10001]: Table not found encrypted_table_n1
-FAILED: SemanticException [Error 10001]: Table not found encrypted_table_n1
+FAILED: SemanticException [Error 10001]: Table not found default.encrypted_table_n1
```

Comment:
it's good to see that it's getting more consistend; but why do we see this 3 times? :D 
I don't fully understand whats going on here....FAILED during a clientpositive test...why is that okay? :dagger: 
(this is clearly outside of the scope of this change)